cvat-ui/src/components/tasks-page/empty-list.tsx

```diff
@@ -31,6 +31,10 @@ export default function EmptyListComponent(): JSX.Element {
             <Row type='flex' justify='center' align='middle'>
                 <Col>
                     <Link to='/tasks/create'>create a new task</Link>
+                    <br />
+                    <Text type='secondary'>or try to</Text>
+                    <br />
+                    <Link to='/projects/create'>create a new project</Link>
```

Comment:
It seems I was wrong about <br /> because understood wrong the previous solution.
Instead of <br /> we just need a couple of additional spaces:

Have you tried something like that?
```
<Link to='/tasks/create'>create a new task</Link>
<Text type='secondary'> or try to </Text>
<Link to='/projects/create'>create a new project</Link>
```

Reply:
Fixed